pkg/apis/recommendation/v1alpha1/types.go

```diff
@@ -93,28 +87,20 @@ const (
 	AlgorithmPercentile Algorithm = "percentile"
 )
 
-// ContainerResourcePolicy controls how computes the recommended
-// resources for a specific container.
+// ContainerResourcePolicy controls how computes the recommended resources for a specific container.
```

Comment:
containerResourcePolicy provides the policy for recommender to manager the given container resource, including the container name, resource kind, usage  buffer etc.